### PR TITLE
chore(all): some lint fixes

### PR DESCRIPTION
**Pull request by mperdomo-bc**
Mostly removal of unused imports and some reordering in files. A few changes to old React Class components without conversion because that would take more time.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/index.tsx [hunk 1]
```diff
@@ -225,7 +225,7 @@ const MakeOffer: React.FC<Props> = (props) => {
           <Value>
             <Field
               name='expirationMinutes'
-              onChange={(days: any) => {
+              onChange={() => {
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/index.tsx:228]: This function here does nothing... we should check if that is OK

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MarkForSale/index.tsx [hunk 2]
```diff
@@ -389,8 +389,7 @@ const MarkForSale: React.FC<Props> = (props) => {
                 <Value>
                   <Field
                     name='expirationMinutes'
-                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                    onChange={(minutes: any) => {
+                    onChange={() => {
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MarkForSale/index.tsx:392]: This function here does nothing... we should check if that is OK

**Review by TheLeoB — APPROVED** [review]
Thanks for doing this. This all seems ok, but we should certainly spot check/test things on staging, make sure there are no unintended side effects

**Comment by mperdomo-bc** [comment]
Got superseded by other working prs

**Pull request closed without merging**